src/notation/qml/MuseScore/NotationScene/internal/TextStylePopup.qml

```diff
@@ -0,0 +1,389 @@
+                            iconRole: IconCode.TEXT_ALIGN_BOTTOM,
+                            typeRole: TextTypes.FONT_ALIGN_V_BOTTOM,
+                            title:qsTrc("inspector", "Align bottom"),
+                            description: qsTrc("inspector", "Align bottom edge of text to reference point")
+                        },
+                        {
+                            iconRole: IconCode.TEXT_ALIGN_BASELINE,
+                            typeRole: TextTypes.FONT_ALIGN_V_BASELINE,
+                            title: qsTrc("inspector", "Align baseline"),
+                            description: qsTrc("inspector", "Align baseline of text to reference point")
+                        }
+                    ]
+
+                    delegate: FlatRadioButton {
+                        toolTipTitle: modelData.title
+                        toolTipDescription: modelData.description
+
+                        width: 30
+                        transparent: true
+
+                        iconCode: modelData.iconRole
+                        checked: root.model && !root.model.verticalAlignment.isUndefined && (root.model.verticalAlignment.value === modelData.typeRole)
+                        onToggled: {
+                            root.model.verticalAlignment.value = modelData.typeRole
+                        }
+                    }
+                }
+            }
+
+            //Column -> 2 | Row -> 2
+            RowLayout {
+
+                //Text subScript and superScript buttons.
+                RadioButtonGroup {
+                    id: subscriptOptionsButtonList
+                    height: 30
+
+                    model: [
+                        { iconRole: IconCode.TEXT_SUBSCRIPT, typeRole: TextTypes.TEXT_SUBSCRIPT_BOTTOM, titleRole: qsTrc("inspector", "Subscript") },
+                        { iconRole: IconCode.TEXT_SUPERSCRIPT, typeRole: TextTypes.TEXT_SUBSCRIPT_TOP, titleRole: qsTrc("inspector", "Superscript") }
+                    ]
+
+                    delegate: FlatRadioButton {
+
+                        width: 30
+                        transparent: true
+
+                        iconCode: modelData["iconRole"]
+                        checked: root.model && !root.model.textScriptAlignment.isUndefined ? root.model.textScriptAlignment.value === modelData["typeRole"]
+                                                                                           : false
+                        onClicked: {
+                            if (root.model.textScriptAlignment.value === modelData["typeRole"]) {
+                                root.model.textScriptAlignment.value = TextTypes.TEXT_SUBSCRIPT_NORMAL
+                            } else {
+                                root.model.textScriptAlignment.value = modelData["typeRole"]
+                            }
+                        }
+                    }
+                }
+
+                //Line Spacing Label
+                Text {
```

Comment:
Please use `StyledTextLabel`, which automatically has the correct font and color. 

(If some property should be different in this case from what `StyledTextLabel` does by default, you can still override specific properties, since `StyledTextLabel` is derived from `Text`.)

Also, for a text label is it clear enough what it is, so you don't need to add a comment for it, and also not an `id` unless that `id` needs to be referenced somewhere.